README.md

```diff
@@ -156,79 +149,96 @@ LATERAL FLATTEN(OUTER => TRUE, INPUT => chains_table."raw_data_column":"restaura
 
 ```LookML
 
+view: JSON_TABLE { 
+  sql_table_name: RESTAURANT_DETAILS ;;
 
-
-view: chains_table { 
-  sql_table_name: chains_table ;;
-
-  dimension: city {
-    description: "City"
+  dimension: country {
```

Comment:
What's the sorting that's applied? Can we create consistently the same result with the same input? Either by keeping the sorting in the JSON (probably the most logical from a user's perspective) or by alphabethical sorting? It's interesting that every commit creates such huge differences.

Reply:
Dimensions were initially stored in an unordered set, I'll sort the dimensions based on their names.